core/include/detray/utils/inspectors.hpp

```diff
@@ -142,7 +142,6 @@ struct object_tracer {
                                        const point3_t &pos,
                                        const vector3_t &dir,
                                        const char * /*message*/) {
-
```

Comment:
The PR is already a monster. Hopefully not many of such unintended changes got in... :thinking:

Reply:
I hope not. This is probably a leftover from rebasing. Removed